R/generatePartialDependence.R

```diff
@@ -186,6 +124,9 @@ generatePartialDependenceData = function(obj, input, features,
     assertDataFrame(data, col.names = "unique", min.rows = 1L, min.cols = length(obj$features) + length(td$target))
     assertSetEqual(colnames(data), c(obj$features, td$target), ordered = FALSE)
   }
+  
+  if (is.na(n[2]))
```

Comment:
What's n[2] here? Any way to do this without a magic number?

Reply:
n[2] is the number of points to sample from the training data to average over. that should be in the documentation for n but will double check

Reply:
Maybe pull that out into a separate variable to make it more obvious what's going on?

Reply:
i think it makes sense as is as both elements have the same function. it is a resolution parameter